crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -66,45 +66,30 @@ pub struct AlgorithmV1 {
     /// The D component of the PID control for the DA gas price
     da_d_factor: i64,
     /// The average profit over the last `avg_window` blocks
-    avg_profit: i64,
-    /// The number of blocks to consider when calculating the average profit
-    avg_window: u32,
+    last_profit: i64,
+    /// the previous profit
+    last_last_profit: i64,
```

Comment:
I find the naming a bit confusing here. How about something like:

```suggestion
    /// The average profit over the second last `avg_window` blocks
    second_last_profit: i64,
```